src/pages/data-statistics/DataStatistics.js

```diff
@@ -115,7 +115,8 @@ class DataStatistics extends Page {
             }
 
             if (
-                userInvitationsResponse.hasOwnProperty(
+                Object.hasOwnProperty.call(
+                    userInvitationsResponse,
```

Comment:
Why is this necessary?

Reply:
If it is, a comment would be useful

Reply:
This is to please the eslint rule https://eslint.org/docs/rules/no-prototype-builtins